### PR TITLE
chore (closes #49): added basic ESLint support

### LINKED ISSUE #49: Enable ESLint
It will be nice to add ESLint to this project.

It is not that big to benefit from ES6, so I don't think we need to bring in additional transpilation step. But having ESLint checks will ensure consistent coding standards for this repo.

Rules may extends `eslint-config-recommended` and use `eslint-env=node`.

### PR DESCRIPTION
**Pull request by bhovhannes**

### CODE REVIEW

**Comment by bhovhannes** [comment]
@blacksonic, can you please have a look on this one?

File: test/loader.spec.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-env mocha */
```
Comment (blacksonic) [inline test/loader.spec.js]: No, it is completely fine, to put it into one configuration file.
Adding comments does not scale if we will have multiple spec files.

Comment (bhovhannes) [inline test/loader.spec.js]: @blacksonic , done

**Comment by blacksonic** [comment]
LGTM

**Pull request merged**